tx.spec

```diff
@@ -0,0 +1,36 @@
+# -*- mode: python -*-
+
+block_cipher = None
+
+
+a = Analysis(['tx'],
+             pathex=['C:\\code\\transifex\\transifex-client'],
```

Comment:
I have also removed pathext=[], not sure what it use for, but if there is error, PyInstaller will output the error with full path using pathext. The executable itself runs fine if I omitted pathext